src/main/java/teammates/common/util/TimeHelper.java

```diff
@@ -553,17 +553,19 @@ public static LocalDateTime parseLocalDateTime(String dateTimeString, String pat
     }
 
     /**
-     * Parses a `LocalDateTime` object from separated date, hour and minute strings.
+     * Parses a {@code LocalDateTime} object from separated date, hour and minute strings.
+     * Example: date "Tue, 01 Apr, 2014", hour "23", min "59"
      *
-     * <p>required parameter format:
-     * date: dd/MM/yyyy  hour: H   min:m
-     * Example: If date is 01/04/2014, hour is 23, min is 59.
+     * @param date  date in format "EEE, dd MMM, yyyy"
+     * @param hour  hour-of-day (0-23)
+     * @param min   minute-of-hour (0-59)
+     * @return      the parsed {@code LocalDateTime} object, or {@code null} if there are errors
      */
-    public static LocalDateTime parseLocalDateTime(String date, String hour, String min) {
+    public static LocalDateTime parseLocalDateTimeForSessionsForm(String date, String hour, String min) {
```

Comment:
How about `parseLocalDateTimeFromSessionsFormInputs`?

Reply:
I think `parseLocalDateTimeFromSessionsForm` is okay. I tried to look to the formatting methods (their inverses) for guidance, but right now they're not consistent; we have `formatDateForSessionsForm` and `formatDateTimeForSessions`. I will standardize the method names when addressing #8645. We should probably standardize each formatting string and their purpose at some point in the future for consistency's sake.

Reply:
I guess `parseLocalDateTimeForSessionsForm` is fine too.

> I will standardize the method names when addressing #8645. We should probably standardize each formatting string and their purpose at some point in the future for consistency's sake.

Sounds good 👍